src/yb/yql/pgwrapper/pg_wrapper.cc

```diff
@@ -371,6 +371,11 @@ Status PgWrapper::Start() {
   }
 
   pg_proc_.emplace(postgres_executable, argv);
+  vector<string> ld_library_path {
+    GetPostgresLibPath(),
+    GetPostgresThirdPartyLibPath()
+  };
+  pg_proc_->SetEnv("LD_LIBRARY_PATH", boost::join(ld_library_path, ":"));
```

Comment:
This is required for the `postgres_fdw` extension to load. Without this, the following happens:

```
yugabyte=# create extension postgres_fdw;
ERROR:  could not load library "/home/yb/postgres/lib/postgres_fdw.so": libpq.so.5: cannot open shared object file: No such file or directory
```